cluster-scope/base/core/namespaces/peribolos-as-a-service/namespace.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: peribolos-as-a-service
+    annotations:
+        openshift.io/display-name: Peribolos Service
+        openshift.io/requester: peribolos
+        op1st/project-owner: open-services-group
```

Comment:
group name here should match the one being created above `peribolos-as-a-service`